src/main/java/ru/mambetpages/controller/PersonalProfileController.java

```diff
@@ -78,7 +82,45 @@ public ResponseEntity<?> login(@RequestBody LoginDto login) {
         }
     }
 
-    @DeleteMapping
+    @DeleteMapping("sessions")
     @Operation(summary = "Выход из профиля")
     public void logout() {}
+
+    @GetMapping("{profileId}/articles")
```

Comment:
i suggest using {profile-id}, in this case @PathVariable UUID profileId should be changed to @PathVariable(name ="profile-id") UUID profileId or so

Reply:
Or just @PathVariable("profile-id")